packages/core/contracts/data-verification-mechanism/implementation/VotingV2.sol

```diff
@@ -493,10 +493,10 @@ contract VotingV2 is Staker, OracleInterface, OracleAncillaryInterface, OracleGo
      ****************************************/
 
     /**
-     * @notice Gets the queries that are being voted on this round.
-     * @return pendingRequests array containing identifiers of type PendingRequestAncillary.
+     * @notice Gets the requests that are being voted on this round.
+     * @return pendingRequests array containing identifiers of type PendingRequestAncillaryAugmented.
      */
-    function getPendingRequests() external view override returns (PendingRequestAncillaryAugmented[] memory) {
+    function getPendingRequests() public view override returns (PendingRequestAncillaryAugmented[] memory) {
```

Comment:
Is this function still useful? Why not just remove view and add processResolvablePriceRequests() as the first line?

Reply:
You are right @mrice32 , there was no need to have 2 functions for this. Fixed in the latest commit. Thank you!